.github/workflows/build-and-deploy.yml

```diff
@@ -174,7 +173,7 @@ jobs:
           if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
             DEPLOYMENT_MATRIX="{ 'environment': ['${{ github.event.inputs.environment }}'] }"
           else
-            DEPLOYMENT_MATRIX="{ 'environment': ['dev', 'test', 'preprod'] }"
+            DEPLOYMENT_MATRIX="{ 'environment': ['dev', 'preprod'] }"
```

Comment:
Remove preprod if successfully migrated
Make production depend on the aks nonprod deploy job